reg/drone/service/actuator/esc/Status.0.1.uavcan

```diff
@@ -0,0 +1,21 @@
+# Detailed ESC status published at a low rate.
+# The publication period should be consistent across the group.
+
+uint8 MAX_PUBLICATION_PERIOD = 1  # [second]
```

Comment:
why 1hz?

Reply:
Indeed, I think it might be better to make it asynchronous in case of an error. We've got a periodic heartbeat as well right>?

Reply:
I think you are reading it as `MIN_PUBLICATION_PERIOD` but it is actually the opposite. The frequency cannot be lower than 1 Hz; while the limit can be changed, it should be defined because subscribers need to be aware of the worst-case data update interval.